api/src/main/java9/org/reactivestreams/FlowAdapters.java

```diff
@@ -157,12 +158,12 @@ private FlowAdapters() {
     static final class FlowToReactiveSubscription implements Flow.Subscription {
         final org.reactivestreams.Subscription reactiveStreams;
 
-        public FlowToReactiveSubscription(org.reactivestreams.Subscription reactive) {
+        public FlowToReactiveSubscription(final org.reactivestreams.Subscription reactive) {
```

Comment:
Other than personal preference, in classes you PRed, we don't really do this style wise in this project...